package.json

```diff
@@ -40,6 +40,11 @@
     "url": "https://github.com/openmrs/openmrs-esm-login/issues"
   },
   "dependencies": {
+    "@carbon/colors": "^10.19.0",
+    "@ibm/plex": "^5.1.3",
```

Comment:
We only need the stylesheet of this, right? Any reason for using this stylesheet?

If you ask me we should rely on "pure" carbon and either duplicate (copy) or reimplement derived styles / components.

Reply:
I discovered Eric used ibm/ipex  at https://app.zeplin.io/project/5f7223cfda10f94d67cec6d0/screen/600805a8aca8ba86e8566ed3 for styling  but am removing it as well